src/setupWorker/setupWorker.ts

```diff
@@ -111,5 +133,10 @@ export function setupWorker(
         ...nextHandlers,
       )
     },
+    recorder: {
```

Comment:
A conceptual question: can a `recorder` be a request handler?

With it being a request handler we get a few things baked-in:

1. Request interceptions. Request handler will receive all unhandled requests prior to its positioning in the list of handlers. This position dependency may be used as a feature to filter out certain requests you don't wish to record.
1. Short-circuit. Since `recorder` always performs requests as-is, it can return them short-circuiting the request handlers lookup. There's no need for an extra modification of `handleRequestWith`.

Reply:
`recorder` as `handler` could be great. My first idea was to add in this way but I had to create all handlers for every method and I was not sure if it was the right thing. It could remain a property of the worker but we can, under the hood, add some handlers instead of updating `handleRequestWith`.

Reply:
This functionality can be embedded into the devtools directly, with MSW only exposing some hooks to listen to. However, it's a good incremental step to introduce the recorder feature gradually, first without the devtool, and then with it. 

If that doesn't look like a lot of trouble, could you try `recorder` as a request handler? I'm extremely curious what difference that would produce.

Reply:
I have updated the code with the hybrid approach. In this way the `recorder` is somethign that can't be confused with the handler, but at the same time we reuse the handler approach to make the recorder works

Reply:
Using only as `handler` like you have done above, how the user then could access to the recorder requests?